server/common/models/widget.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "widget",
+  "plural": "widgets",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "options": {
+    "validateUpsert": true
+  },
+  "properties": {
+    "widName": {
```

Comment:
Lets use a consistent naming convention. Please update all code to use a consistent naming.
- Decide if you want to use short forms or full words
- Decide between camel case or underscore